include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1579,7 +1594,8 @@ namespace internal
         task_info.initial_setup_blocks_tasks(subdomain_boundary_cells,
                                              renumbering,
                                              irregular_cells);
-        task_info.guess_block_size(dof_info[0].dofs_per_cell[0]);
+        task_info.guess_block_size(
+          dof_info[first_dof_handler_index].dofs_per_cell[0]);
```

Comment:
Since there are many use cases of `dof_info[first_dof_handler_index]` in the following code, should we create an alias like
```suggestion
        const internal::MatrixFreeFunctions::DoFInfo &dof_info_hp = dof_info[first_dof_handler_index];
        task_info.guess_block_size(
          dof_info_hp.dofs_per_cell[0]);
```
and use it here and the respective functions below?